NuRadioMC/utilities/medium_base.py

```diff
@@ -375,6 +376,192 @@ def _compute_default_ice_model_radiopropa(self):
             
 
 
+class IceModelBirefringence(IceModelSimple):
+    """
+    predefined ice model (to inherit from) including different indieces of refraction for differnt directions (birefringence)
+    """
+    def __init__(self,
+                 exp_model,
+                 bir_model,
+                 n_ice,
+                 delta_n,
+                 z_0,
+                 z_shift
+                 ):
+
+        """
+        initiaion of a simple exponential ice model
+
+        The bottom defined here is a boundary condition used in simulations and
+        should always be defined. Note: it is not the same as reflective bottom.
+        The latter can be added using the `add_reflective_layer` function.
+
+        The z_shift is a variable introduced to be able to shift the exponential
+        up or down along the z direction. For simple models this is almost never
+        but it is used to construct more complex ice models which rely on exp.
+        profiles also
+
+        Parameters
+        ----------
+        exp_model:    pointer
+                        exponential parametrisation of the density effect on the refractive index
+        bir_model:    str
+                        choice for the birefringence model, 'southpole_A', 'southpole_B', ...
+        n_ice:  float, dimensionless
+                refractive index of the deep bulk ice
+        delta_n:  float, NuRadio length units
+                  difference between n_ice and the refractive index
+                  of the snow at the surface
+        z_0:  float, NuRadio length units
+              scale depth of the exponential
+        z_shift:  float, NuRadio length units
+                  up or down shift od the exponential profile
+        """
+
+        super().__init__(n_ice, delta_n, z_0, z_shift)
+        
+        self.exp_model = exp_model
+        self.bir_model = bir_model
+
+        self.f1 = interpolate.UnivariateSpline._from_tck(self.bir_model[0])
+        self.f2 = interpolate.UnivariateSpline._from_tck(self.bir_model[1])
+        self.f3 = interpolate.UnivariateSpline._from_tck(self.bir_model[2])
+
+        self.comp = self.exp_model.get_index_of_refraction(np.array([0, 0, -2500]))
+
+
+    def get_index_of_refraction(self, position):
```

Comment:
this function is already implemented in the base class from which you inherit.

Reply:
Done.